bundles/org.openhab.binding.meteostick/src/main/java/org/openhab/binding/meteostick/internal/MeteostickBindingConstants.java

```diff
@@ -51,6 +51,7 @@ public class MeteostickBindingConstants {
     public static final String PARAMETER_CHANNEL = "channel";
     public static final String PARAMETER_SPOON = "spoon";
     public static final String PARAMETER_SPOON_DEFAULT = "0.254";
+    public static final String PARAMETER_WINDVANE = "deltawinddir";
```

Comment:
Please use camelCase: https://www.openhab.org/docs/developer/guidelines.html#naming-convention
```suggestion
    public static final String PARAMETER_WINDVANE = "deltaWindDirection";
```

Also update the readme and xml.